standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/DirectSqlUpdateStat.java

```diff
@@ -647,6 +653,12 @@ public long getNextCSIdForMPartitionColumnStatistics(long numStats) throws MetaE
       jdoConn = pm.getDataStoreConnection();
       dbConn = (Connection) (jdoConn.getNativeConnection());
 
+      if (sqlGenerator.getDbProduct().isMYSQL()) {
```

Comment:
Is there any way to filter and set only if the query contains quotes?

Reply:
that will be costlier than setting the value in DB. As we dont know how many sql statements are getting executed in side a connection.